scopes/component/new-component-helper/new-component-helper.main.runtime.ts

```diff
@@ -31,16 +32,44 @@ export class NewComponentHelperMain {
     return ComponentID.fromObject({ name: fullComponentName }, scope);
   }
 
+  /**
+   * Synchronously checks if the given directory contains any files.
+   */
+  private dirContainsFiles(dirPath: string): boolean {
+    if (!fs.existsSync(dirPath)) return false;
+
+    const contents = fs.readdirSync(dirPath);
+    for (const item of contents) {
+      if (fs.statSync(path.join(dirPath, item)).isFile()) {
```

Comment:
Do you care if it's a file or dir?

Reply:
If it's a file, then it's probably a component folder and I don't want to create it there
There's probably a better way to check it, but I didn't want to use any async methods
I know it's something we check anyway when coming to create the component

Reply:
thinking of it again, we don't have to check for that because if it has files (and there's a component there) then it will fail anyway when checking for it